### DISCUSSION TITLE
Setting up Azure CID dashboard for multiple Azure accounts

### DISCUSSION THREAD

**Discussion by dheerajsharma14** [aws-solutions-library-samples/cloud-intelligence-dashboards-framework#814] [Q&A]
Hi, 

I am deploying the Cloud intelligence Dashboard for Azure following the guidelines mentioned in the below sample:

https://github.com/aws-samples/aws-data-pipelines-for-azure-storage?tab=readme-ov-file#cloud-intelligence-dashboard-for-azure

However, my situation is that I have multiple Azure Accounts with their own Azure ADs and each having multiple subscriptions in them. Currently, as per the design, it seems that the template works for only a "single Azure Account with an Azure AD and multiple subscriptions in it". 

I understand that I can duplicate the dashboard for multiple Azure Management accounts but then that defeats the purpose of centralized reporting as I will have to open up multiple dashboards for my multiple Azure accounts. I cannot consolidate those multiple Azure accounts under one because they all belong to the child entities of the parent company.

It seems that the AWS CID Dashboard does support multiple AWS Organization/Management accounts as per the architecture and we were able to set them up. But, Azure cannot find a way.

So, my question is can we support multiple Azure Management Accounts each with their own Azure AD in a single Azure CID Dashboard? 

Any guidance on it will help.

<img width="1845" alt="cidazure-midlevel-grey" src="https://github.com/aws-samples/aws-cudos-framework-deployment/assets/18235572/d7b9488b-238e-4c5b-a1e2-737628ea95c0">

**Comment by iakov-gan** [comment]
You can deliver all billing files to the same storage account at the Azure end. This way a single dashboard can be used and they can filter by billing account name. 

feel free to contact CID Azure team as described here -> https://catalog.workshops.aws/cidforazure/en-US#feedback
